default-config/config

```diff
@@ -34,6 +34,16 @@ InfoStoreAdd terminal xterm
 # This is used for "Run Command" and the Meta+Space key-binding
 InfoStoreAdd runcmd "dmenu_run -nb '#2b4e5e'"
 
+# DesktopConfiguration
+#
+# Global: This is the default setting. In Global mode all screens share desks and pages
+#
+# Per-Desktop: This mode every screen have its own deks and pages. An Pager would be required for each screen.
```

Comment:
You mean `per-monitor` here.

I would say: `In this mode evey screen has its own desk and pages.    It's recommended that `FvwmPager` is configured to use the `Monitor` configuration option, so that only windows on that specific monitor are shown.

Reply:
You are correct. I'll fix it now